pennylane/templates/layers/basic_entangler.py

```diff
@@ -110,18 +81,12 @@ def circuit(weights):
 
         **Parameter initialization function**
 
-        The :mod:`~pennylane.init` module has two parameter initialization functions, ``basic_entangler_layers_normal``
-        and ``basic_entangler_layers_uniform``.
+        A random numpy weights array can be generated using the static methods
+        `BasicEntanglerLayers.weights_normal` and `BasicEntanglerLayers.weights_uniform`.
```

Comment:
Could we link to those methods? Something like
```rst
:meth:`~.BasicEntanglerLayers.weights_normal`
```
so people can click and it'll take to the docstring?